report/generator/flow-report-assets.js

```diff
@@ -6,11 +6,10 @@
 'use strict';
 
 const fs = require('fs');
-const {LH_ROOT} = require('../root.js');
 
 /* eslint-disable max-len */
-const FLOW_REPORT_TEMPLATE = fs.readFileSync(`${LH_ROOT}/flow-report/assets/standalone-flow-template.html`, 'utf8');
-const FLOW_REPORT_JAVASCRIPT = fs.readFileSync(`${LH_ROOT}/dist/report/flow.js`, 'utf8');
+const FLOW_REPORT_TEMPLATE = fs.readFileSync(`${__dirname}/../../flow-report/assets/standalone-flow-template.html`, 'utf8');
+const FLOW_REPORT_JAVASCRIPT = fs.readFileSync(`${__dirname}/../../dist/report/flow.js`, 'utf8');
```

Comment:
@connorjclark sorry, these had to move off using `root.js` because that would expose generator code to all the dependencies of the root `tsconfig.json` (really `root.js` is a leaf file as well for dependencies, but tsc isn't smart enough to allow a simple case like that).

The report assets won't ever have that many paths to look up, so we could use `import.meta.url` just for these files, we could make a `root.js` copy for generator code, or we could (eventually?) do the "shared" directory idea from above and put `root.js` in there. This seemed ok for now since `report-assets.js` still uses `__dirname` too.